### ISSUE TITLE
Could not load your font, installed it globally as admin

### ISSUE THREAD

**Issue by SpoonMcForky** [Piotrekol/StreamCompanion#300]

**Comment by SpoonMcForky** [comment]
Also why does nothing actually uninstall when running the installer 🤔

**Comment by SpoonMcForky** [comment]
also loading a font with a number in it completely crashes and corrupts osu

**Comment by Piotrekol** [comment]
Take a chill pill and provide details regarding your first issue.

For 2 others do the same, but in separate github issues (as a side note, there is no way for SC to "corrupt" osu!)

**Comment by SpoonMcForky** [comment]
Yeah sorry, had a rough day and this not working just set me over I guess. I gave as much detail as I could; I opened SC and then I closed it, installed a font, opened it up again, selected the font in SC for the overlay, and launched osu. The first time it worked, and the font was loaded correctly, but when I went to change the size/position, it could no longer load the font. I uninstalled the font and SC numerous times, including deleting all of the files.
